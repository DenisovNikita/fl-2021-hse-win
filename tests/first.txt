0 to 1 by 1 0
0 to 3 by 1 1
1 to 4 by 1 0
1 to 4 by 1 1
2 to 2 by 1 0
2 to 3 by 1 1
3 to 3 by 1 0
3 to 2 by 1 1
4 to 4 by 1 0
4 to 4 by 1 1
terminal 2
1
2

