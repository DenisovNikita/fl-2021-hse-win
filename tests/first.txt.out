LexToken(NUM,0,1,0)
LexToken(TO,'to',1,2)
LexToken(NUM,1,1,5)
LexToken(BY,'by',1,7)
LexToken(NUM,0,1,10)
LexToken(NUM,0,2,12)
LexToken(TO,'to',2,14)
LexToken(NUM,3,2,17)
LexToken(BY,'by',2,19)
LexToken(NUM,1,2,22)
LexToken(NUM,1,3,24)
LexToken(TO,'to',3,26)
LexToken(NUM,4,3,29)
LexToken(BY,'by',3,31)
LexToken(NUM,0,3,34)
LexToken(NUM,1,4,36)
LexToken(TO,'to',4,38)
LexToken(NUM,4,4,41)
LexToken(BY,'by',4,43)
LexToken(NUM,1,4,46)
LexToken(NUM,2,5,48)
LexToken(TO,'to',5,50)
LexToken(NUM,2,5,53)
LexToken(BY,'by',5,55)
LexToken(NUM,0,5,58)
LexToken(NUM,2,6,60)
LexToken(TO,'to',6,62)
LexToken(NUM,3,6,65)
LexToken(BY,'by',6,67)
LexToken(NUM,1,6,70)
LexToken(NUM,3,7,72)
LexToken(TO,'to',7,74)
LexToken(NUM,3,7,77)
LexToken(BY,'by',7,79)
LexToken(NUM,0,7,82)
LexToken(NUM,3,8,84)
LexToken(TO,'to',8,86)
LexToken(NUM,2,8,89)
LexToken(BY,'by',8,91)
LexToken(NUM,1,8,94)
LexToken(NUM,4,9,96)
LexToken(TO,'to',9,98)
LexToken(NUM,4,9,101)
LexToken(BY,'by',9,103)
LexToken(NUM,0,9,106)
LexToken(NUM,4,10,108)
LexToken(TO,'to',10,110)
LexToken(NUM,4,10,113)
LexToken(BY,'by',10,115)
LexToken(NUM,1,10,118)
LexToken(TERMINAL,'terminal',11,120)
LexToken(NUM,2,11,129)
LexToken(NUM,1,12,131)
LexToken(NUM,2,13,133)
